### PR TITLE
Feature/merge HecDss7

### PR DESCRIPTION
**Pull request by adamkorynta**

### CODE REVIEW

File: wrims_v2/wreslparser/README.md
Comment (rma-tom) [inline wrims_v2/wreslparser/README.md:1]: Minor -- file says "evaluator" where it should say "wreslparser"

Comment (adamkorynta) [reply wrims_v2/wreslparser/README.md:1]: that can get edited in a follow-up PR as that typo was present in the Feature/HecDss7 branch

File: wrims_v2/wrims_v2/src/test/test_heclib7/Test_heclib7.java
Comment (rma-tom) [inline wrims_v2/wrims_v2/src/test/test_heclib7/Test_heclib7.java:1]: This file and others in wrims_v2/wrims_v2/src/test/test_heclib7 belong in 
wrims-core/src/test/test_heclib7 in the devops structure.

Comment (adamkorynta) [reply wrims_v2/wrims_v2/src/test/test_heclib7/Test_heclib7.java:1]: none of the tests under wrims_v2/wrims_v2 were moved under wrims-core

Comment (rma-tom) [reply wrims_v2/wrims_v2/src/test/test_heclib7/Test_heclib7.java:1]: I see. There's a test with the same name in the devops branch in rmanet. We'll need to review the tests we want to bring over, but I don't think it's an issue for this PR

**Review by rma-tom — APPROVED** [review]
Comments on individual files are resolved. I don't have any remaining issues.

**Review by hxiedwr — APPROVED** [review]
I think the changes are correct.

**Pull request merged**